inference-engine/src/mkldnn_plugin/emitters/jit_load_store_emitters.cpp

```diff
@@ -156,16 +156,27 @@ void jit_load_emitter::load_bytes(const Vmm &vmm, const Xbyak::Reg64 &reg, int o
         return ptr[reg + offset + bytes_offset * sizeof(int8_t)];
     };
 
-    if (is_zmm && (load_size != 64) && mayiuse(cpu::x64::avx512_core)) {
-        uint64_t mask = 1;
-        mask = (mask << load_size) - mask;
-        h->mov(Reg64(aux_gpr_idxs[0]), mask);
-        h->kmovq(k_mask, Reg64(aux_gpr_idxs[0]));
-        h->vmovdqu8(zmm | k_mask | T_z, addr(0));
+    if (is_zmm) {
+        switch (load_size) {
+            case 64:
+                h->uni_vmovdqu(zmm, addr(0));
+                break;
+            case 32:
+                h->uni_vmovdqu(ymm, addr(0));
+                break;
+            case 16:
+                h->uni_vmovdqu(xmm, addr(0));
+                break;
+            default:
+                uint64_t mask = 1;
+                mask = (mask << load_size) - mask;
+                h->mov(Reg64(aux_gpr_idxs[0]), mask);
+                h->kmovq(k_mask, Reg64(aux_gpr_idxs[0]));
+                h->vmovdqu8(zmm | k_mask | T_z, addr(0));
```

Comment:
This instruction is available in AVX512BW only. So we need to preserve `mayiuse(cpu::x64::avx512_core)` check.

Reply:
fixed, thank you!